src/modules/python/pbs_hooks/PBS_power.PY

```diff
@@ -308,17 +308,14 @@ if this_event.type == pbs.PERIODIC:
                 sleep_node_list = fd.read().split(',')
         except IOError:
             raise Exception("I/O error reading node file")
```

Comment:
I'm still unsure why you catch IOError and reraise an exception.  Why not just remove the try/except block and let the IOError go through uncaught.

Reply:
I changed the block to reject the hook.